### ISSUE TITLE
Digoxin use

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10675]
**Preferred term label:**
Digoxin use

**Synonyms**
Digitalis, Lanoxin

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Digoxin toxicity

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001138
- label: Digoxin exposure
- parents: [Medication history (HP:4000112)](https://hpo.jax.org/app/browse/term/HP:4000112)
- pmid: PMID:29262029
- def: History of having been treateed with digoxin.
- comment: Digoxin is a cardiac glycoside derived from the foxglove plant (digitalis species). It has inotropic effects and is utilized in the management of systolic dysfunction in patients with congestive heart failure (CHF) and as an atrioventricular nodal blocking agent for managing atrial tachydysrhythmias. Gastrointestinal upset is the most common symptom of digoxin toxicity. Patients also may report visual symptoms, which classically present as a yellow-green discoloration, and cardiovascular symptoms, such as palpitations, dyspnea, and syncope.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10675
- synonyms: Digitalis exposure(NONE)|Lanoxin exposure(NONE)
